packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -25,20 +26,22 @@ type ForeachOptions = {
 // eslint-disable-next-line arca/no-default-export
 export default (clipanion: any, pluginConfiguration: PluginConfiguration) => clipanion
 
-  .command(`workspaces foreach run <command> [...args] [-v,--verbose] [-p,--parallel] [-i,--interlaced] [-j,--jobs JOBS] [--topological] [--topological-dev] [--include WORKSPACES...] [--exclude WORKSPACES...]`)
+  .command(`workspaces foreach <command> [...args] [-v,--verbose] [-p,--parallel] [-i,--interlaced] [-j,--jobs JOBS] [--topological] [--topological-dev] [--all] [--include WORKSPACES...] [--exclude WORKSPACES...]`)
```

Comment:
Of note, we initially chose to make the `run` keyword explicit to leave the room later to add more verbs (for example `yarn workspaces foreach publish`). That said it's not necessarily incompatible, so I'm fine if we try it this way.

cc @deini

Reply:
A bit late to the party but I'm fine if we try it this way too. We can revisit if adding more verbs causes confusion.